pytorch_blade/benchmark/TorchBench/test_torch_bench.sh

```diff
@@ -34,19 +34,25 @@ ln -s $benchmark_repo_dir torchbenchmark
 # setup benchmark env
 export DISC_EXPERIMENTAL_SPECULATION_TLP_ENHANCE=true \
     DISC_CPU_LARGE_CONCAT_NUM_OPERANDS=4 DISC_CPU_ENABLE_EAGER_TRANSPOSE_FUSION=1 \
-    OMP_NUM_THREADS=1 TORCHBENCH_ATOL=1e-2 TORCHBENCH_RTOL=1e-2
+    TORCHBENCH_ATOL=1e-2 TORCHBENCH_RTOL=1e-2
 
 config_file=blade_$1_$2.yaml
 bench_target=$2
 
 if [ $1 == "cpu" ]
 then
-    # 4 cores
-    export GOMP_CPU_AFFINITY="2-5"
+    if [ ! -n "$3" ]
+    then
+        export OMP_NUM_THREADS=1 GOMP_CPU_AFFINITY=4
+    else
+        # eg: bash test_torch_bench.sh cpu tiny 32 "0-63:2"
+        export OMP_NUM_THREADS=$3 GOMP_CPU_AFFINITY=$4
```

Comment:
not enough if only limiting OMP threads since the framework (e.g. torch/TF) may have its own thread pool as well. maybe be use `taskset` as instead?

Reply:
fixed